x-pack/plugins/security_solution/public/detections/components/detection_engine_filters/detection_engine_filters.tsx

```diff
@@ -13,18 +13,18 @@ import { ControlGroupRenderer } from '@kbn/controls-plugin/public';
 import type { AlertFilterControlsProps } from '@kbn/alerts-ui-shared/src/alert_filter_controls';
 import { AlertFilterControls } from '@kbn/alerts-ui-shared/src/alert_filter_controls';
 import { useHistory } from 'react-router-dom';
+import type { DataViewSpec } from '@kbn/data-plugin/common';
 import { useKibana } from '../../../common/lib/kibana';
 import { DEFAULT_DETECTION_PAGE_FILTERS } from '../../../../common/constants';
 import { URL_PARAM_KEY } from '../../../common/hooks/use_url_state';
 import { useSpaceId } from '../../../common/hooks/use_space_id';
-import type { SecuritySolutionDataViewBase } from '../../../common/types';
 import { SECURITY_ALERT_DATA_VIEW } from '../../constants';
 
 export type DetectionEngineFiltersProps = Pick<
   AlertFilterControlsProps,
   'filters' | 'onFiltersChange' | 'query' | 'timeRange' | 'onInit'
 > & {
-  indexPattern?: SecuritySolutionDataViewBase;
+  indexPattern?: DataViewSpec;
```

Comment:
I know this was changed before your PR but it could be a nice time to update this name from `indexPattern` as well, it's only used in one other file looks like so not many extra changes needed